### PR TITLE
67 emissivity

### PR DESCRIPTION
**Pull request by DavidJRitter904**
## Describe your changes

As discussed in Meeting 04.09.2024:

- [x] Add emissivity data as object type with separated emissivity and temperature values.

**Some questions to ask**:

- What is the change?
- What does it fix?
- Is this a bug fix or a feature? Does it break any existing functionality or force me to update to a new version?
- How has it been tested?

## Issue ticket number and link

Closes issue [#67](https://github.com/asam-ev/OpenMATERIAL/issues/67).

## Mention a member

Add @mentions of the person or team responsible for reviewing the proposed changes.

## Checklist before requesting a review

- [x] I have performed a self-review of my code/documentation.
- [x] My changes generate no new warnings during the documentation generation.

### CODE REVIEW

**Comment by ClemensLinnhoff** [comment]
Do we need the possibility to have multiple value pairs of temperature and emissivity? In this implementation, only two fixed values are possible.

**Comment by DavidJRitter904** [comment]
> Do we need the possibility to have multiple value pairs of temperature and emissivity? In this implementation, only two fixed values are possible.

It gives the possibility to add your own emissivity data. 
We would anyway provide the table as in the legacy branch at T=300.0 K with a fixed emissivity value then.

**Comment by DavidJRitter904** [comment]
A further problem of the table given in the legacy for emissivity is, that the value is not fixed for all materials but includes ranges.

[Legacy emissivity table](https://www.engineeringtoolbox.com/emissivity-coefficients-d_447.html)

**Comment by ClemensLinnhoff** [comment]
> Do we need the possibility to have multiple value pairs of temperature and emissivity? In this implementation, only two fixed values are possible.

Group decision: No lookup table with temperature, angle and wavelength for now. This could be added in a later release.

**Comment by ClemensLinnhoff** [comment]
> We use the **_hemispherical total emissivity_** which considers full emission over all wavelengths, directions and polarization for a given particular temperature:   ε(T)

Could you add that to the description?
And please also add the source field analog to #82.

**Comment by DavidJRitter904** [comment]
@ClemensLinnhoff I have updated again the branch and resolved the new merge conflicts. 
The emissivity data should be here now, including the elasticity and density data already pulled from the main. 👍

**Review by ClemensLinnhoff — APPROVED** [review]

**Comment by drsftx73** [comment]
@ClemensLinnhoff with Emissive materials, a custom shader (or modifications to a simulator's 'standard' shader) will be required for this to look correct. Also additional textures, point/spot lights will be required as well. Is this outside of our scope?

**Comment by ClemensLinnhoff** [comment]
> @ClemensLinnhoff with Emissive materials, a custom shader (or modifications to a simulator's 'standard' shader) will be required for this to look correct. Also additional textures, point/spot lights will be required as well. Is this outside of our scope?

This is not about conventional light sources. This is about thermal radiation.

File: examples/example_material.xomp [hunk 1]
```diff
@@ -15,6 +15,11 @@
         "surface_height_rms": 0.8,
         "surface_correlation_length": 1.0
     },
+    "emissivity_data": {
+        "emissivity": 0.07,
```
Comment (ClemensLinnhoff) [inline examples/example_material.xomp:19]:
```suggestion
        "emissivity_coefficient": 0.07,
```

File: schemas/material_schema.json [hunk 2]
```diff
@@ -82,6 +82,29 @@
             "surface_correlation_length"
           ]
         },
+        "emissivity_data": {
+          "type": "object",
+          "description": "Emissivity data of the material",
+          "properties": {
+            "emissivity": {
```
Comment (ClemensLinnhoff) [inline schemas/material_schema.json:89]:
```suggestion
            "emissivity_coefficient": {
```

**Review by ClemensLinnhoff — CHANGES_REQUESTED** [review]

File: examples/example_material.xomp (hunk 1 above)
Comment (ClemensLinnhoff) [reply examples/example_material.xomp:19]: It was discussed in the material group, that emissivity_coefficient would be a bit more accurate.

**Comment by KimuraDIVP** [comment]
I just want to confirm that this "emmisivity" is for the far infrared rays sensor, right?

**Comment by ClemensLinnhoff** [comment]
Needs to be adapted to new naming convention, after #110 is merged,

**Comment by ClemensLinnhoff** [comment]
> I just want to confirm that this "emmisivity" is for the far infrared rays sensor, right?

Mainly for infrared cameras.

**Review by KimuraDIVP — APPROVED** [review]

**Review by ClemensLinnhoff — APPROVED** [review]

**Pull request merged**